scripts/repos/apm.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I would recommend moving this file to a higher directory with a self-descriptive name *Aragon Apps Fetched For Aragen*. It could be turned into `.js` that exports JSON so it can include a nice long comment on top explaining its purpose and methodology

> Open a PR to add your app aragen's CI, etc